### PR TITLE
Dasha

### PR DESCRIPTION
**Pull request by DanyaBu**
Willing to say hi to guys starting this awesome course

### CODE REVIEW

**Comment by artem-trubin** [comment]
Hi! Can you please use Markdown syntax to format text, not HTML?

**Comment by A-Ostrovnyy** [comment]
@DanyaBu avoid using HTML tags in markdown use [this resource](https://codingnconcepts.com/markdown/markdown-vs-html/) to convert your HTML to markdown.

**Comment by DanyaBu** [comment]
> Hi! Can you please use Markdown syntax to format text, not HTML?

Done
Thank you in advance)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+
+I am studyng a FE course from Kottans
+My github is [DanyaBu](https://github.com/DanyaBu)
+
+Good luck to everybody who is also beginning this challenging journey!
+
 #HI!!!!
 
 Have a good day, guys!!!!
+master
```
Comment (A-Ostrovnyy) [inline README.md:17]: I think this must be deleted)

File: README.md [hunk 2]
```diff
@@ -21,6 +35,7 @@ I hope this project will be of interest to each of us.
 My name is Stas, I'm begginner , have some knowledge in HTML , CSS.
 I've a big wish to become frontend developer.
 My repo :(https://github.com/stass777/kottans-frontend)
+master
```
Comment (A-Ostrovnyy) [inline README.md:38]: I think this must be deleted too)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One more step)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
+Dasha
+Dasha
```
Comment (A-Ostrovnyy) [inline README.md:2]: Those branch names must be deleted too

**Pull request closed without merging**